### DISCUSSION TITLE
Bridge deck

### DISCUSSION THREAD

**Discussion by kaasym** [Universite-Gustave-Eiffel/NoiseModelling#665] [Q&A]
Hi, is it possible to take into account the reflectivity (impermeability) of the floor and roof for buildings? I'm trying to model traffic on the bridge and I haven't figured out how to ensure the impermeability of the bridge deck (I'm modeling it as a building). Thanks. Jirka

**Comment by nicolas-f** [comment]
Hi,

You could use the script Enrich_DEM_with_road in order to put the bridge under the road as a terrain.

**Comment by kaasym** [comment]
It is complicated by the fact that under the overpass there is another road leading across.

**Comment by kaasym** [comment]
Wouldn't it be possible to enter a bridge deck as a locally reflective ground in 3D (x, y, z)? I understand that ground is entered only as x, y coordinates, but wouldn't it be possible to extend the calculation by the dimension z?

**Reply by pierromond** [reply]
unluckly no for the moment. You can do an aerial road I think (we never tried) but it will be without reflective road. You can double the traffic on this section if you want 3dB more, but I do not see other easy solution for this moment. In cnosso, nothing is specified on this case.
